trunk-recorder/main.cc

```diff
@@ -954,6 +954,75 @@ void handle_call(TrunkMessage message, System *sys) {
 
 }
 
+void clear_stale_moto_patches(System *sys){
```

Comment:
I'm honestly not 100% sure.  I used this naming convention since that's how its referred to in p25_parser.cc:
`BOOST_LOG_TRIVIAL(debug) << "tsbk00\tMoto Patch Add \tsg: " << sg << "\tga1: " << ga1 << "\tga2: " << ga2 << "\tga3: " << ga3;`
From what I've read, it's my understanding that patching isn't defined in the P25 standard itself and that Moto and Harris use different methods.  My local system is Motorola, so that's what I've been testing with.  I haven't found anything in P25_parser.cc or elsewhere that's given me much insight into how other systems handle it.

Reply:
If I'm interpreting [this post](http://forums.radioreference.com/threads/p25-patch-following.311780/post-2376651) correctly, it looks like Harris patching may use a different set of messages that would need to be handled by p25_parser.cc.  However I think the end result would be pretty much the same as far as what data would get stored in the System class.  The main difference is that the Harris system may use a unique TGID (SAID) from a pool of IDs that's reserved for patching rather than using one of the patched TGIDs as the "supergroup/patch" ID.  But as far as trunk-recorder is concerned it's still just a list of TGIDs.

Since it looks like a different type of Message content may be needed for Harris (something other than the sg,ga1,ga2,ga3 used by Moto), I think it may make sense to keep the 'moto' terminology in the MOTO_PATCH_ADD message type and handling functions.  However, I think it's likely that the Harris message handling functions (if/when added) could likely still use the same data structure to hold the patch information in the System class.

Reply:
Gotcha - thanks for digging into this.

Reply:
Let's change this to `clear_stale_patches`. This function should be able to handle whatever formats we have to use.

Reply:
> Let's change this to clear_stale_patches. This function should be able to handle whatever formats we have to use.

Got it, make sense.